packages/lodestar/src/sync/regular/naive/naive.ts

```diff
@@ -62,11 +62,11 @@ export class NaiveRegularSync extends (EventEmitter as { new(): RegularSyncEvent
     if (headSlot >= currentSlot) {
       this.logger.info(`Regular Sync: node is up to date, headSlot=${headSlot}`);
       this.emit("syncCompleted");
+      await this.stop();
       return;
     }
     this.currentTarget = headSlot;
     this.logger.verbose(`Regular Sync: Current slot at start: ${currentSlot}`);
-    this.targetSlotRangeSource = pushable<ISlotRange>();
```

Comment:
I would leave this actually, since we should also add restarting regular sync if we encounter peer with higher slot, in which we would all start again and it would fail because of closed source.